xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -426,6 +498,114 @@ private String processRouteConfig(RouteConfiguration config) {
     return null;
   }
 
+  /**
+   * Handles EDS response, which contains a list of ClusterLoadAssignment messages. Each
+   * ClusterLoadAssignment message contains information for endpoints in a cluster, grouped
+   * by localities. The response is NACKed if contains invalid information for gRPC's usage.
+   * Otherwise, an ACK request is sent to management server. Information in ClusterLoadAssignment
+   * messages are saved in local cache, in case of corresponding endpoint watchers are registered
```

Comment:
Can you be more specific about local cache? Is it `{@link #clusterNamesToEndpointUpdates}`?

Reply:
I would not say that in method specification, it should not mention implementation state.